### ISSUE TITLE
How do i give range to my date field, i am not able to do it from the UI

### ISSUE THREAD

**Issue by hidayathBR** [kevinchappell/formBuilder#1525]
### Environment Details:

 - formBuilder Version: 
 - Browser: 
 - OS: 

### Screenshot - *(optional)*
![image](https://github.com/kevinchappell/formBuilder/assets/153521501/ad4a4d3b-979f-434a-b352-8d40b172d8a2)

**Comment by hidayathBR** [comment]
Yes there is a Min and Max field to set the range. But it dosent have a format, the average user might think it as a regular textbox.

**Comment by lucasnetau** [comment]
Your screenshot would suggest you have disabled the min/max attributes via config

**Comment by lucasnetau** [comment]
PRs welcome to improve the interface.

Two options would be to 
  1) add placeholders showing the expected format yyyy-mm-dd,
  2) need to convert value, min and max inputs to match the Date/local datetime/time field.
